app/services/provider_interface/hesa_data_export.rb

```diff
@@ -0,0 +1,164 @@
+require 'csv'
+
+module ProviderInterface
+  class HesaDataExport
+    STATUSES = {
+      'awaiting_decision' => ApplicationStateChange::DECISION_PENDING_STATUSES,
+      'offered' => ApplicationStateChange::OFFERED_STATES,
+      'accepted' => ApplicationStateChange::ACCEPTED_STATES,
+      'conditions_met' => [:recruited],
+      'deferred' => [:offer_deferred],
+      'declined' => [:offer_declined],
+      'rejected' => [:offer_rejected],
+      'rejected_automatically' => [],
+      'application_withdrawn' => [:withdrawn],
+      'offer_withdrawn' => [:offer_withdrawn],
+    }.freeze
+
+    def initialize(export_options: {})
```

Comment:
It looks like we only ever use `provider_ids` from this hash? Would it be better to simply have these as named params as it's extra ceremony for the caller, eg https://github.com/DFE-Digital/apply-for-teacher-training/pull/3135/files#diff-bdff2c50fb9f23305fcf23f9de8fd2382f263023b7abf9be1b7ecd4ec7d0cd5dR8

Reply:
Yes for sure, this is a relic of attempting the full-fat version with status and recruitment years filters, we decided not to include these initially. I can strip this back.

Reply:
@duncanjbrown I've simplified this to only pass `provider_ids` and have removed the redundant form class.